_posts/2018-11-19-announcing-riff-0-2-0.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: single
+title: "Announcing riff v0.2.0"
+header:
+  overlay_image: /images/geese.jpg
+excerpt:
+  With buildpacks everywhere, smarter detection, and updated Knative support
+permalink: /blog/announcing-riff-0-2-0/
+---
+
+We are happy to announce the release of [riff v0.2.0](https://github.com/projectriff/riff/releases/tag/v0.2.0). Thank you once again, all riff and Knative contributors.
+
+The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/riff/releases/tag/v0.2.0) on GitHub. Please follow one of the [getting started](/docs) guides, to create a new cluster on GKE or minikube.
+
+Notable changes in this release include:
+- All builds now use [Buildpacks](https://buildpacks.io/)
+- No more special case builds of images starting with `dev.local`
+- `riff function create`
+  - no `<invoker>` argument before the `<name>` argument
+  - new optional `--invoker` flag
+  - `--builder` flag removed
+- `riff function build` has been renamed `riff function update`
+- `riff service revise` has been renamed `riff service update`
+- `riff namespace init` has a new `--no-secret` flag
+
+### Buildpacks everywhere!
+This release extends the use of buildpacks across all of our currently supported invokers: Java, JavaScript, and Command. 
+
+Here is a map of the buildpack-related repos on Github.
```

Comment:
A contributor might care, a user probably won't and will stop reading

Reply:
think our readers will appreciate the extra detail on this topic.